### PR TITLE
[FIX] Add `message.replies` undefined property check in the new message template

### LINKED ISSUE #26639: "Uncaught Error" in some group chats with new message template in Desktop App

### PR DESCRIPTION
**Pull request by krenuser**
## Proposed changes (including videos or screenshots)

[FIX] fixed new message template issue for some rooms, which is related to `message.replies` property usage without pre-check.

## Issue(s)

This fixes #26639 

## Steps to test or reproduce

_(Mentioned on issue page)_

### CODE REVIEW

File: apps/meteor/client/views/room/MessageList/components/MessageContent.tsx [hunk 1]
```diff
@@ -82,14 +82,14 @@ const MessageContent: FC<{ message: IMessage; sequential: boolean; subscription?
 				<ThreadMetric
 					openThread={openThread(message._id)}
 					counter={message.tcount}
-					following={Boolean(mineUid && message?.replies.indexOf(mineUid) > -1)}
+					following={Boolean(mineUid && message && message.replies && message?.replies.indexOf(mineUid) > -1)}
```
Comment (debdutdeb) [inline apps/meteor/client/views/room/MessageList/components/MessageContent.tsx:85]:
```suggestion
					following={Boolean(mineUid && message?.replies?.indexOf(mineUid) > -1)}
```

File: apps/meteor/client/views/room/MessageList/components/MessageContent.tsx [hunk 2]
```diff
@@ -82,14 +82,14 @@ const MessageContent: FC<{ message: IMessage; sequential: boolean; subscription?
...
 					mid={message._id}
 					rid={message.rid}
 					lm={message.tlm}
 					unread={Boolean(subscription?.tunread?.includes(message._id))}
 					mention={Boolean(subscription?.tunreadUser?.includes(message._id))}
 					all={Boolean(subscription?.tunreadGroup?.includes(message._id))}
-					participants={message?.replies.length}
+					participants={message && message.replies ? message?.replies.length : 0}
```
Comment (debdutdeb) [inline apps/meteor/client/views/room/MessageList/components/MessageContent.tsx:92]:
```suggestion
					participants={message?.replies?.length || 0}
```

**Review by debdutdeb — CHANGES_REQUESTED** [review]

**Review by debdutdeb — APPROVED** [review]
Lgtm

**Comment by krenuser** [comment]
Can't understand what has failed during build job. 
Is it my fault? Should I fix something?

**Comment by filipemarins** [comment]
We have a [pr with unit test](https://github.com/RocketChat/Rocket.Chat/pull/26652).

**Pull request closed without merging**